src/puppetlabs/puppetdb/cli/benchmark.clj

```diff
@@ -292,36 +300,90 @@
 (def required-cli-options
   [:config])
 
+(defn validate-options
```

Comment:
We definitely need more of this in benchmark.clj, it's good to see you adding that for the new params. When I was trying out the patch, I found NumberFormatExceptions when run interval and numhosts was missing. Wrote it up as a separate ticket: https://tickets.puppetlabs.com/browse/PDB-1268